src/init/register_configure_agent.sh

```diff
@@ -94,12 +92,16 @@ add_adress_block() {
     fi
 
     # Write the client configuration block
-    for i in ${SET_ADDRESSES};
+    for i in ${!ADDRESSES[@]};
```

Comment:
Shellcheck report:
```
    for i in ${!ADDRESSES[@]};
             ^-- [SC2068](https://www.shellcheck.net/wiki/SC2068) (error): Double quote array expansions to avoid re-splitting elements.
```
What do you think?

Reply:
Done [b04ac96](https://github.com/wazuh/wazuh/pull/15038/commits/b04ac96d39822d256022cdf5fde42d57f5e043b9)